client/ViewDeck.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ConfirmedButton from './FormComponents/ConfirmedButton';
+import DeckSummary from './DeckSummary';
+import Panel from './SiteComponents/Panel';
+
+class ViewDeck extends React.Component {
```

Comment:
I'm fairly ambivalent on this component. It does help separate another large chunk out of Decks rendering wise. It also ensures that each component deals with things at the same level of abstraction - Decks only deals with events on decks (selecting, editing, deleting) whereas ViewDeck deals with the lower level UI concerns (clicking on the edit and delete button and translating them to the proper deck-oriented events).

It's a subtle distinction that I think makes the overall code cleaner and more focused - but it's also a bunch of boilerplate code. So overall I think it's a win but not sure if it's such a slam-dunk win to justify itself. 🤷‍♀️